testgres/__init__.py

```diff
@@ -24,3 +24,7 @@
     get_bin_path, \
     get_pg_config, \
     get_pg_version
+
+from .standby import \
+    First, \
```

Comment:
IMHO those names are too generic to be exported at top level.

Reply:
I think names are ok but should be used with the of module, like `standby.First`

Reply:
@ildus I agree.